### ISSUE TITLE
aria-label and button text mismatch

### ISSUE THREAD

**Issue by irv** [tomickigrzegorz/show-more#81]
Following on from #75

WCAG 2.2 wants to `aria-label` value to be the same as the button text (e.g. "Show more")

ref https://www.w3.org/WAI/WCAG22/Techniques/failures/F96

It seems as simple a fix as the below?

```patch
index 450faff..880159e 100644
--- a/static/js/showMore.js
+++ b/static/js/showMore.js
@@ -175,7 +175,7 @@ var ShowMore = (function () {
         btnClassAppend
       } = _ref;
       const typeAria = this._checkExp ? less || "" : more || "";
-      const label = this._checkExp ? "collapse" : "expand";
+      const label = this._checkExp ? less : more;
       const expanded = this._checkExp ? true : false;
       const button = createElement("button");
       button.className = btnClassAppend == null ? btnClass : btnClass + " " + btnClassAppend;
```

**Comment by tomickigrzegorz** [comment]
Thanks,
I'll have a moment to check, I'm not an expert in WCAG 2.2 ;)
